tree/tree/inc/ROOT/RFriendInfo.hxx

```diff
@@ -19,9 +19,9 @@
 #define ROOT_RFRIENDINFO_H
 
 #include <ROOT/RStringView.hxx>
+#include <Rtypes.h> // Long64_t
```

Comment:
Is it better to use `Rtypes` or `RTypescore` in this case?

Reply:
$ rg 'Rtypes\.h' tree/dataframe | wc -l
11
$ rg 'Rtypescore\.h' tree/dataframe | wc -l
0

:man_shrugging:

Reply:
Yeah the spelling is not the most brilliant here :sweat_smile: 

```
$: grep -r 'RtypesCore.h' tree/dataframe | wc -l
23
```

So maybe in a separate PR we should decide on which of the two headers is better to use everywhere? There must be some difference that makes one useful in some cases where the other isn't.

Reply:
> must be some difference that makes one useful in some cases where the other isn't.

One is older than the other.  One (the same one) include the others.  If `RtypesCore.h` is sufficient, it should be preferred.  Essentially `Rtypes.h` is `RtypesCore.h` + `ClassDef` declarations.